### PR TITLE
Start using Asset Registry 

### PR DESCRIPTION
**Pull request by ranjanan**
This uses Asset Registry for the katex css and js 

cc: @shashi

### CODE REVIEW

File: src/output.jl [hunk 1]
```diff
@@ -9,8 +17,9 @@ Render `txt` in LaTeX using KaTeX. Backslashes need to be escaped:
 function latex(txt)
    (txt isa Observable) || (txt = Observable(txt))
    w = Scope(imports=[
-           "/pkg/InteractBase/npm/node_modules/katex/dist/katex.min.js",
-           "/pkg/InteractBase/npm/node_modules/katex/dist/katex.min.css"])
+                      katex_min_js,
+                      katrx_min_css
```
Comment (piever) [inline src/output.jl]: typo

**Comment by piever** [comment]
Thanks! I would be curious to know what's the current state of Asset Registry: does it work reliably in all supported environment already? Say Jupyter lab / notebook, Juno, Blink, Mux?

File: src/output.jl [hunk 2]
```diff
@@ -1,5 +1,13 @@
 using WebIO, JSExpr
 
+const katex_min_js = joinpath(@__DIR__, "..", "assets",
```
[the remark this answers is not in the document]
Comment (ranjanan) [reply src/output.jl]: No, when you send them to a WebIO scope, WebIO internally registers them using Asset Registry.

**Comment by ranjanan** [comment]
Yes it does! It works well on Blink and Mux. I think it works on IJulia as well, though @shashi will know for sure.

File: src/output.jl (hunk 1 above)
Comment (ranjanan) [reply src/output.jl]: Thanks, fixed.

File: src/output.jl (hunk 2 above)
[the remark this answers is not in the document]
Comment (ranjanan) [reply src/output.jl]: > `import!(w::Scope, InteractBase.katex_min_js)`

Ah, I think WebIO needs a patch for that. I think @shashi just implemented it in https://github.com/JuliaGizmos/WebIO.jl/pull/141. 

> IIUC, we actually do not even need to load AssetRegistry as all we're doing is passing a string to a Scope

Yes, that is correct. In that case I guess we don't even need the `using AssetRegistry` or the change to the REQUIRE. Let me just test that.

[the remark this answers is not in the document]
Comment (ranjanan) [reply src/output.jl]: Oh so it turns out you don't need it in REQUIRE. I updated the PR.

**Comment by shashi** [comment]
It works on Jupyter as well.

**Pull request merged**